pkg/cmd/project/list/list.go

```diff
@@ -138,26 +138,29 @@ func buildURL(config listConfig) (string, error) {
 	return url, nil
 }
 
-func filterProjects(nodes []queries.Project, config listConfig) []queries.Project {
-	projects := make([]queries.Project, 0, len(nodes))
-	for _, p := range nodes {
+func filterProjects(nodes queries.Projects, config listConfig) queries.Projects {
+	projects := queries.Projects{
+		Nodes:      make([]queries.Project, 0, len(nodes.Nodes)),
+		TotalCount: nodes.TotalCount,
+	}
+	for _, p := range nodes.Nodes {
 		if !config.opts.closed && p.Closed {
 			continue
 		}
-		projects = append(projects, p)
+		projects.Nodes = append(projects.Nodes, p)
 	}
 	return projects
 }
```

Comment:
Just some small nits on the naming here, I know this was already in place, but think we can improve it while we are here.
```
func filterProjects(projects queries.Projects, config listConfig) queries.Projects {
	filtered := queries.Projects{
		Nodes:      make([]queries.Project, 0, len(projects.Nodes)),
		TotalCount: projects.TotalCount,
	}
	for _, project := range projects.Nodes {
		if !config.opts.closed && project.Closed {
			continue
		}
		filtered.Nodes = append(filtered.Nodes, project)
	}
	return filtered
}
```